dcos-labs/ui-kit
# D2IQ-67847: add pointer caret to Popover

Relates to D2IQ-67847

## Testing

- Confirm that the Popover component has a caret pointing at the target (just like the Tooltip component)
- Ensure the style refactoring didn't break any Tooltip styles

## Trade-offs



## Dependencies



## Screenshots
![Screen Shot 2020-06-22 at 2 00 21 PM](https://user-images.githubusercontent.com/2313998/85320971-e85eb580-b491-11ea-8aeb-e6497d63152d.png)

## Checklist for PR creator

- [ ] If any new components were added, there are exported from `packages/index.ts`
- [ ] If this PR is associated with a JIRA, it is mentioned in commit message footer ("Closes …")
- [ ] If this PR contains breaking changes, is stated in commit message body ("BREAKING CHANGE: …")
- [x] Info for applicable sections above is provided
